### ISSUE TITLE
Provide a documentation about design choices & openapi-core limitiations

### ISSUE THREAD

**Issue by playpauseandstop** [playpauseandstop/rororo#28]
Some `openapi-core` issues, such as,

- Incomplete support of free-form objects
- Avoiding validation error data in `OpenAPIMediaTypeError` exceptions
- Missing support of operation security

resulted in specific implementation details, which should be covered in documentation to provide better understanding why `rororo` need to fix given (and other) issues with `openapi-core` or other tools.

**Comment by playpauseandstop** [comment]
Need to review this issue after #35

**Comment by playpauseandstop** [comment]
All of issues fixed in after upgrade to latest `openapi-core` #35

Closing as `wontfix`
